CMakeLists.txt

```diff
@@ -56,7 +56,6 @@ endif()
 # Options
 option(BUILD_SOX "Build libsox statically" OFF)
 option(BUILD_KALDI "Build kaldi statically" ON)
-option(BUILD_TRANSDUCER "Enable transducer" OFF)
```

Comment:
I think we can keep this but default to `ON`. Keeping this allows faster build time with `BUILD_TRANSDUCER=OFF`. Thoughts?

Reply:
Indeed, given that this is still a prototype, let's keep the option in, and leave the switch of default to later.